spec/docker/exec_spec.rb

```diff
@@ -138,6 +138,17 @@
       end
     end
 
+    context 'when :wait set with some values' do
+      subject {
+        described_class.create('Container' => container.id, 'Cmd' => %w[date])
+      }
+      after { container.kill!.remove }
+
+      it 'returns empty stdout/stderr messages with exitcode', :vcr do
+        expect(subject.start!(:wait => 100)).to eq([[],[], 0])
+      end
```

Comment:
Can you also add a test case for when the command is something like `["/bin/bash","-lc","sleep 2"]` with a timeout of 1? If VCR doesn't seem to want to record that, don't worry about adding it.